src/io/flutter/view/InspectorPanel.java

```diff
@@ -192,7 +195,7 @@ public void onAppChanged() {
 
   @Nullable
```

Comment:
This should now be `@NotNull` I believe.

Reply:
done